### DISCUSSION TITLE
Listing - Owner

### DISCUSSION THREAD

**Discussion by JFOC** [inspireui/support#5245] [New Feature/Ideas Requests]
Need an owner ability and manage the listing from the apps for the owner as Listeo has this feature, also the registration for owner

**Comment by ahmedkaazab** [comment]
I hope they can create such an option , working in web-view for add listing is not so good

**Comment by hoangphuc1103** [comment]
Thank you for your contribution; the owner registration feature is now available and supported in the Fluxstore Listing app 🎉
